experiments/models.py

```diff
@@ -105,9 +105,9 @@ def is_open(self):
 
     def uses_groups(self):
         """
-        An experiment is designed to use groups when it has more than one group.
+        An experiment is designed to use groups when it has at least one group.
         """
-        return self.targetgroup_set.count() > 1
+        return self.targetgroup_set.count() > 0
```

Comment:
Cool, this makes life easier I think! As there is 1 group by default, so one can use the session api!